fuzzing/repositories.bzl

```diff
@@ -18,12 +18,15 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load("//fuzzing/private/oss_fuzz:repository.bzl", "oss_fuzz_repository")
 
-def rules_fuzzing_dependencies(oss_fuzz = True, honggfuzz = True):
+def rules_fuzzing_dependencies(oss_fuzz = True, honggfuzz = True, jazzer = True):
```

Comment:
Existing (C++) users of this library upgrading to this revision will now automatically pull the Jazzer dependencies in their workspace. Since this is adding an unused dep to all existing users, how about turning it off by default? Depending on future usage information, we can decide to turn it back on.

Reply:
Sounds good.